build-image/builder.sh

```diff
@@ -43,6 +46,22 @@ if [ "$(cat /tmp/go-build/build_master)" == "${short_commit}" ]; then
   exit 130
 fi
 
+log "Verifying tag signatures..."
+for tag in ${tags}; do
+  if ( LANG=C git tag --verify ${tag} 2>&1 | grep "Good signature" ); then
+    LANG=C git tag --verify ${tag} 2>&1 | grep "gpg:" > /tmp/go-build/.signature_${tag}
```

Comment:
Ah, with the “grepping the raw GnuPG output” approach you *could* use gobuilder WoT to check for impersonators.  But that makes it hard to use gobuilder for folks outside its WoT.  If you are auto-trusting in gobuilder, you should probably use the `%G?` format outlined [here][1] to save yourself the trouble of keeping up with GnuPG's output (although that's likely pretty stable ;).

[1]: http://mikegerwitz.com/papers/git-horror-story#automate